DependencyInjection/Configuration.php

```diff
@@ -34,6 +34,14 @@ public function getConfigTreeBuilder()
                     ->prototype('scalar')
                     ->end()
                 ->end()
+                ->arrayNode('extractor')
```

Comment:
this config is currently dead code, as you remove all usage of this config from the DI extension